doc/install/install-gromacs.md

```diff
@@ -0,0 +1,33 @@
+# Install GROMACS with DeepMD
+## 1. Patch source code of GROMACS 
+Download source code of a supported gromacs version from https://manual.gromacs.org. Run the following command:
```

Comment:
If you only support Gromacs 2020.2, then provide the download link to that specific version.